src/js/components/Carousel/Carousel.js

```diff
@@ -81,9 +81,11 @@ class Carousel extends Component {
   };
 
   render() {
-    const { children, fill, focus, theme, ...rest } = this.props;
+    const { children, fill, focus, theme, controls, ...rest } = this.props;
```

Comment:
```suggestion
    const { children, controls, fill, focus, theme, ...rest } = this.props;
```
we are trying to keep list orders in an alphabetical way :)